C-Summary/index.html

```diff
@@ -22,8 +22,24 @@ <h2>Great</h2>
         <p class="transparent">You scored higher than 65% of the people who have taken these tests.</p>
       </div>
     </section>
-
+    <section class="right-section">
+      <div class="summary-overall">
+        <h3>Summary</h3>
+        <p class="block-red"><img src="./assets/images/icon-reaction.svg" alt="Image of lightning">Reaction <span class="bold-number"><b>80</b><span class="not-bold"> / 100</span></span></p>
```

Comment:
> Nice work Ivan. See how this can be published on GitHub to give you a review of the published page. https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site

Regarding this, I'm not sure what branch I should publish and if it publishes all other branches off of it like challenges>right-panel. Also I don't know if I should set the source to /docs or /root.